ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/SchemaInfo.java

```diff
@@ -36,6 +37,23 @@
   private final SqlBaseType type;
   private final ImmutableList<FieldInfo> fields;
   private final SchemaInfo memberSchema;
+  private final Map<String, Object> fieldProperties;
+
+  @JsonCreator
```

Comment:
according to https://stackoverflow.com/a/15931483/2258040 you can only have one constructor with `@JsonCreator` annotated. I think it should be this one - but we should test to see what happens if we try to deserialize a JSON object without `fieldProperties`. I'm pretty sure it'll work (it'll create an empty map) but if not we need to make `fieldProperties` optional.

Reply:
Yeah ConsoleTest proves it's good